build.sbt

```diff
@@ -0,0 +1,34 @@
+import Dependencies._
+
+ThisBuild / scalaVersion     := "2.13.8"
+ThisBuild / version          := "0.1.0-SNAPSHOT"
+ThisBuild / organization     := "uk.gov.nationalarchives"
+ThisBuild / organizationName := "The National Archives"
+ThisBuild / startYear := Some(2023)
+ThisBuild / licenses := Seq("MIT" -> url("https://opensource.org/licenses/MIT"))
+ThisBuild / description := "JMS4S Request-Reply Library"
+
+/* IMPORTANT NOTE >>>
+ * Until further notice you will need to sbt publishLocal the JMS4S library from the combined-features branch
+ * of this repository https://github.com/rwalpole/jms4s
+ */
+
+lazy val root = (project in file("."))
+  .enablePlugins(AutomateHeaderPlugin)
+  .settings(
+    name := "jms4s-request-reply",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %% "cats-core" % "2.9.0",
+      "org.typelevel" %% "cats-effect" % "3.4.4",
+      "org.typelevel" %% "cats-effect-kernel" % "3.4.4",
+      //"org.typelevel" %% "log4cats-slf4j" % "2.5.0",
+      "org.typelevel" %% "log4cats-core" % "2.5.0",
+      "dev.fpinbo" %% "jms4s-active-mq-artemis" % "0.0.1-350e4fd-SNAPSHOT",
```

Comment:
I can't build this - `publishLocal` on the `combined-features` branch of your fork of the `jms4s` library gives me a different hash. I got around this by adding an explicit version to the build of the `jms4s` library, [here](https://github.com/rwalpole/jms4s/pull/1)

Reply:
Then I changed this dependency:
```
      "dev.fpinbo" %% "jms4s-active-mq-artemis" % "0.0.1-COMBINED-SNAPSHOT",
      "dev.fpinbo" %% "jms4s-simple-queue-service" % "0.0.1-COMBINED-SNAPSHOT",
```

Reply:
I just took whatever version was generated from `publishLocal`; in my case:

```
/Users/tna/.ivy2/local/dev.fpinbo
├── jms4s-active-mq-artemis_2.13
│   └── 0.0.1-2d6d83c
├── jms4s-ibm-mq_2.13
│   └── 0.0.1-2d6d83c
├── jms4s-simple-queue-service_2.13
│   └── 0.0.1-2d6d83c
└── jms4s_2.13
    └── 0.0.1-2d6d83c
```

But yes: same sort of fix.